### PR TITLE
Fix extension hanging during Firebase Init

### PR DESCRIPTION
**Pull request by hlshen**

### CODE REVIEW

File: firebase-vscode/src/cli.ts [hunk 1]
```diff
@@ -79,12 +79,15 @@ export async function requireAuthWrapper(
   // for subsequent API calls. Warning: this variable takes precedence
   // over Google login tokens and must be removed if a Google
   // account is the current user.
+  console.log("HAROLD: CHECK LOGIN");
```
Comment (rrousselGit) [inline firebase-vscode/src/cli.ts:82]:
```suggestion
```

File: firebase-vscode/src/cli.ts [hunk 2]
```diff
@@ -79,12 +79,15 @@ export async function requireAuthWrapper(
...
   try {
     const optsCopy = currentOptions.value;
     const userEmail = await requireAuth(optsCopy); // client email
     // SetAccessToken is necessary here to ensure that access_tokens are available when:
     // - we are using tokens from configstore (aka those set by firebase login), AND
     // - we are calling CLI code that skips Command (where we normally call this)
+        console.log(currentOptions.value);
```
Comment (rrousselGit) [inline firebase-vscode/src/cli.ts:89]:
```suggestion
```

File: firebase-vscode/src/cli.ts [hunk 3]
```diff
@@ -79,12 +79,15 @@ export async function requireAuthWrapper(
...
+        console.log(userEmail);
```
Comment (rrousselGit) [inline firebase-vscode/src/cli.ts:90]:
```suggestion
```

**Review by rrousselGit — APPROVED** [review]

**Review by joehan — APPROVED** [review]
LGTM after cleaning up the logs

**Pull request merged**